EIPS/eip-1822.md

```diff
@@ -1,9 +1,9 @@
 ---
 eip: 1822
 title: Universal Upgradeable Proxy Standard (UUPS)
-author: Gabriel Barros <gabriel@terminal.co>, Patrick Gallagher <blockchainbuddha@gmail.com>
+author: Gabriel Barros <gbbabarros@gmail.com>, Patrick Gallagher <blockchainbuddha@gmail.com>
```

Comment:
Ah, the problem is that you changed your email address on the author line to one that doesn't match your GitHub address.  This has been a problem in the past so our new policy is that there should always be at least one author in the authors list that has a GitHub handle.
```suggestion
author: Patrick Gallagher (@pi0neerpat), Gabriel Barros <gbbabarros@gmail.com>, Patrick Gallagher <blockchainbuddha@gmail.com>
```
or
```suggestion
author: Patrick Gallagher (@pi0neerpat), Gabriel Barros <gbbabarros@gmail.com>
```